src/Phinx/Db/Adapter/SQLiteAdapter.php

```diff
@@ -204,7 +204,7 @@ public function connect(): void
      */
     public static function getSuffix(array $options): string
     {
-        if ($options['name'] === self::MEMORY) {
+        if (($options['name'] ?? '') === self::MEMORY) {
```

Comment:
This produced a notice when running tests (with `error_reporting = E_ALL` in php.ini)